ipapython/ipaldap.py

```diff
@@ -1068,7 +1069,8 @@ def _connect(self):
 
         return conn
 
-    def simple_bind(self, bind_dn, bind_password, server_controls=None,
+    def simple_bind(self, bind_dn=DN(('cn', 'directory manager')),
+                    bind_password='', server_controls=None,
```

Comment:
I would really prefer if both `bind_dn` and `bind_password` had to be always specified explicitly.